src/PointwiseFunctions/GeneralRelativity/Ricci.cpp

```diff
@@ -9,7 +9,7 @@
 
 template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::aa<DataType, SpatialDim, Frame, Index> compute_ricci(
-    const tnsr::abb<DataType, SpatialDim, Frame, Index>& christoffel_2nd_kind,
+    const tnsr::Abb<DataType, SpatialDim, Frame, Index>& christoffel_2nd_kind,
```

Comment:
Can you apply these bugfixes to the commit where the bug was introduced?

Reply:
With help from Nils (thanks!) after screwing up my git repo, I squashed these commits into one, since they are both for the same purpose.